lib/shelley/test/data/cardano-node-shelley/node.config

```diff
@@ -26,13 +26,13 @@ PBftSignatureThreshold: 1
 
 ApplicationName: cardano-sl
 ApplicationVersion: 1
-LastKnownBlockVersion-Major: 1
+LastKnownBlockVersion-Major: 2
 LastKnownBlockVersion-Minor: 0
 LastKnownBlockVersion-Alt: 0
 
 # Launcher code will submit an update proposal to trigger the hard-fork.
-TestShelleyHardForkAtVersion: 1
-
+TestShelleyHardForkAtEpoch: 1
+TestAllegraHardForkAtEpoch: 2
```

Comment:
The original intention of the `withCluster` was to allow the caller to run code on each era of the chain. Looking now, perhaps the existing doc comment doesn't make it clear, but:

could we still add a note about the fork(s) being triggered automatically in the documentation for `withCluster`?

Reply:
Yes, if we need the wallet to work in two eras then we ought to run some tests in both eras.
But perhaps while the wallet is effectively allegra-only it can be simpler and use the hard fork schedule.